packages/phone-number-privacy/combiner/.env

```diff
@@ -9,4 +9,4 @@ CONTEXT_NAME='alfajores'
 # TODO investigate why these are defined here 
 NODE_OPTIONS='--require ./dist/tracing.js'
 TRACER_ENDPOINT='https://grafana-agent.staging-odis2-centralus.celo-networks-dev.org/api/traces'
-TRACING_SERVICE_NAME='odis-combiner-staging'
+TRACING_SERVICE_NAME='odis-combiner-gen2-staging'
```

Comment:
should this be moved to the context specific .env files?

Reply:
added to context specific .env